bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/dto/SurePetcareBridgeConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.surepetcare.internal.dto;
+
+/**
+ * The {@link SurePetcareBridgeConfiguration} is a container for all the bridge configuration.
+ *
+ * @author Rene Scherer - Initial contribution
+ */
+public class SurePetcareBridgeConfiguration {
+
+    public String username;
+    public String password;
+    public Long refreshIntervalTopology;
+    public Long refreshIntervalStatus;
```

Comment:
makes sense, but how do I get hold of the default value specified in the thing-descriptions xml file?

Reply:
You would have the default value specified in both places. No way around that at the moment. But frankly it shouldn't matter since the default value in the xml should end up overriding the default value specified in the code anyway. So ultimately the default value in the xml will take precedence.

Reply:
done